manifests/03-rbac-role-ns-openshift-config-managed.yaml

```diff
@@ -0,0 +1,21 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: console-operator
```

Comment:
We are assuming the creation of the `namespace` `openshift-config-managed` before this file is applied.  I think its a safe assumption, but just pointing it out.  I'm not sure off the top of my head on the ordering.

Reply:
@spadgett do you know the ordering by any chance?

Reply:
> We are assuming the creation of the `namespace` `openshift-config-managed` before this file is applied. I think its a safe assumption, but just pointing it out. I'm not sure off the top of my head on the ordering.

Looking at the manifests in the release payload here, it shouldn't be an issue:

https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/openshift_console-operator/180/pull-ci-openshift-console-operator-master-e2e-aws/1229/artifacts/release-latest/release-payload-latest/